cli/src/utils.rs

```diff
@@ -71,3 +75,22 @@ impl FileOrUrl {
         }
     }
 }
+
+pub(crate) fn print_docs_with_indent(docs: &[String], indent: usize) -> String {
+    // take at most the first paragraph of documentation, such that it does not get too long.
```

Comment:
I feel like this hsould be reflected in the function name, since it would be easy to be confused at why docs weren't all being printed otherwise, eg `print_first_paragraph_with_indent`